cmd/masonry/masonry_test.go

```diff
@@ -12,19 +12,24 @@ import (
 
 var usage = `
 Usage:
-  masonry [command]
+  masonry [global-options] COMMAND [command-options]
```

Comment:
where are the ``global-options`` documented?

Reply:
@shawndwells In the help/usage outputs for each of the commands. For example, `masonry help diff`